src/_data/authors.yml

```diff
@@ -0,0 +1,145 @@
+anho:
```

Comment:
Are we doing authors in one file or in a separate file? It seems like we have both approaches in the PR

Reply:
Nope. All authors are in one file [_data/authors.yml](https://github.com/nodesagency/blog/blob/feature/implement-design/src/_data/authors.yml) but [_authors](https://github.com/nodesagency/blog/blob/feature/implement-design/src/_authors) folder consists pages for each authors for information like biografie. These pages have layout called [author](https://github.com/nodesagency/blog/blob/feature/implement-design/_config.yml#L21).